VideoLocker/res/drawable-v21/light_selectable_rounded_box_overlay.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ripple xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I don't think I've seen ripple anywhere else in our App. Just making sure, does UI know about it? Are they goog with it? I'm sure you ran it by them, just checking.